services/s3control/src/it/java/software.amazon.awssdk.services.s3control/S3ControlIntegrationTest.java

```diff
@@ -102,6 +104,40 @@ public void getPublicAccessBlock_NoSuchAccount() {
         }
     }
 
+    @Test
+    public void getPublicAccessBlock_NoSuchPublicAccessBlock() {
+        try {
+            client.getPublicAccessBlock(r -> r.accountId(accountId));
+            fail("Expected exception");
+        } catch (S3ControlException e) {
+            assertEquals("NoSuchPublicAccessBlockConfiguration", e.awsErrorDetails().errorCode());
```

Comment:
Can we change this to `assertThat`? And same as all the other usages below? `assertThat` is more readable and flexible, and it's used more often in our code base.

Reply:
I prefer assertThat too. Changed all assertEquals in the file (but did not touch the assertTrue etc)